pre-docs/bolt_installing.md

```diff
@@ -216,3 +216,6 @@ We support using Kerberos over SSH for authentication, but due to license incomp
 gem install net-ssh-krb
 ```
 
+If using `puppet-bolt` packages, the `gem` program can be found
```

Comment:
I think this needs it's own section in installing modules

Reply:
Something similar already exists in https://github.com/puppetlabs/bolt/blob/master/pre-docs/applying_manifest_blocks.md#using-hiera-data-in-a-manifest-block. This seems most relevant when installing Bolt, and we could link back to this section from the others.

Reply:
@katelopresti do you have thoughts on where this should live?